pkg/proxy/proxier.go

```diff
@@ -319,38 +319,43 @@ type Proxier struct {
 	hostIP        net.IP
 }
 
+var (
+	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
```

Comment:
Who is "we" ?

- glog.Errorf("Failed to select a host interface: %v", err)

- glog.Errorf("Failed to initialize iptables: %v", err)

- glog.Errorf("Failed to flush iptables: %v", err)

Why is this one more important?  does someone need to test whether an error they receive is the "not on localhost" error?  Who/where?